spec/controllers/orders_controller_spec.rb

```diff
@@ -683,46 +680,70 @@ def deliver; end
           @submitted_date = 2.days.ago.change(hour: 14, min: 27)
         end
 
-        it "sets the order_detail states to completed because it's in the past" do
-          do_request
-          assigns[:order].order_details.all? { |od| expect(od.state).to eq("complete") }
-        end
+        context "for a reservation-only instrument" do
+          before :each do
+            @instrument = FactoryGirl.create(:setup_instrument,
+                                             facility: @authable,
+                                             facility_account: @facility_account)
+            setup_past_reservation
+          end
 
-        it "sets the order_detail fulfilment dates to the order time" do
-          do_request
-          assigns[:order].order_details.all? do |od|
-            expect(od.fulfilled_at).not_to be_nil
-            expect(od.fulfilled_at).to match_date @reservation.reserve_end_at
+          it "does not set the actual times to the completed reservation times" do
+            do_request
+            expect(@reservation.reload.actual_start_at).to be_nil
+            expect(@reservation.actual_end_at).to be_nil
           end
         end
 
-        it "sets the actual times to the completed reservation times" do
-          do_request
-          expect(@reservation.reload.actual_start_at).to match_date @reservation.reserve_start_at
-          expect(@reservation.actual_end_at).to match_date(@reservation.reserve_start_at + 60.minutes)
-        end
+        context "for a non reservation-only instrument" do
+          before :each do
+            @instrument = FactoryGirl.create(:instrument,
```

Comment:
Can we make these two contexts use the same base factory, either `:instrument` and `:setup_instrument` and use either `no_relay` on `:instrument` or your new `:not_reservation_only` on `:setup_instrument`?